variables.sh

```diff
@@ -9,7 +9,7 @@ if [ "${ALLSKY_VARIABLE_SET}" = "" ]; then
 
 	ALLSKY_VARIABLE_SET="true"	# so we only do the following once
 
-	ME="$(basename "${BASH_SOURCE[0]}")"
+	ME2="$(basename "${BASH_SOURCE[0]}")"
```

Comment:
Why not using ME_VARIABLES ?

Reply:
@AndreasLMeg Only because other files that are source'd in also use $ME2.  I considered making it a "local" variable so I could use the same name, but thought that would be too confusing.